activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Add namespaced version of the `db:migrate:reset` command.
```

Comment:
It's more accurate to reference multiple databases since they aren't namespaced otherwise.

```suggestion
*   Add support for multiple databases to `db:migrate:reset`.
```